### PR TITLE
Fix iOS Identity editor height

### PR DESCRIPTION
**Pull request by tddang-linagora**
## Issue
- https://github.com/linagora/tmail-flutter/issues/3101#issuecomment-2306298330

## Dependencies
- https://github.com/linagora/enough_html_editor/pull/33
- https://github.com/linagora/rich-text-composer/pull/29

## Demo

https://github.com/user-attachments/assets/b5468152-fb83-454d-b2ae-7c23d2346892

**Rich text + lazy loading images**

https://github.com/user-attachments/assets/53d1cb98-761d-45aa-a27f-8e1537dee717

### CODE REVIEW

File: lib/features/identity_creator/presentation/identity_creator_view.dart [hunk 1]
```diff
@@ -549,6 +549,7 @@ class IdentityCreatorView extends GetWidget<IdentityCreatorController>
     return HtmlEditor(
       key: controller.htmlKey,
       minHeight: controller.htmlEditorMinHeight,
+      adjustHeight: true,
```
Comment (dab246) [inline lib/features/identity_creator/presentation/identity_creator_view.dart:552]: `adjustHeight: true` is the default value. So we don't need to add it.

File: lib/features/identity_creator/presentation/identity_creator_view.dart [hunk 2]
```diff
@@ -575,11 +576,13 @@ class IdentityCreatorView extends GetWidget<IdentityCreatorController>
     return Column(children: [
       _buildCheckboxIdentityDefault(context),
       const SizedBox(height: 24),
-      Row(children: [
-        Expanded(child: _buildCancelButton(context)),
-        const SizedBox(width: 12),
-        Expanded(child: _buildSaveButton(context))
-      ])
+      SafeArea(
```
Comment (dab246) [inline lib/features/identity_creator/presentation/identity_creator_view.dart:579]: why we need it?

Comment (tddang-linagora) [reply lib/features/identity_creator/presentation/identity_creator_view.dart:579]: The bottom buttons is overlap with system navigation bar

Comment (dab246) [reply lib/features/identity_creator/presentation/identity_creator_view.dart:579]: Please attach a screen shot of the problem attached to the description.

File: lib/features/identity_creator/presentation/identity_creator_view.dart (hunk 1 above)
Comment (tddang-linagora) [reply lib/features/identity_creator/presentation/identity_creator_view.dart]: Done

**Comment by hoangdat** [comment]
please verify also in `vacation` screen

File: lib/features/identity_creator/presentation/identity_creator_view.dart (hunk 2 above)
Comment (tddang-linagora) [reply lib/features/identity_creator/presentation/identity_creator_view.dart:578]: Simulator Screenshot - iPhone 15 - 2024-08-27 at 15 09 13

**Comment by tddang-linagora** [comment]
> please verify also in `vacation` screen

Simulator Screenshot - iPhone 15 - 2024-08-27 at 15 13 21

File: pubspec.lock [hunk 3]
```diff
@@ -1645,8 +1645,8 @@ packages:
     dependency: "direct main"
     description:
       path: "."
-      ref: main
-      resolved-ref: "4a4ee913fbdec0f98cc1755f31df54fb402d0957"
+      ref: "bugfix/ios-editor-height"
```
Comment (hoangdat) [inline pubspec.lock:1648]: also merge and pump patch version for https://github.com/linagora/rich-text-composer.git

**Comment by hoangdat** [comment]
LGTM

File: pubspec.lock [hunk 4]
```diff
@@ -436,8 +436,8 @@ packages:
     dependency: transitive
     description:
       path: "."
-      ref: master
-      resolved-ref: "55817316c081a363a410fd0c14867be28096790d"
+      ref: "bugfix/ios-editor-height"
```
[the remark this answers is not in the document]
Comment (tddang-linagora) [reply pubspec.lock]: Done

File: pubspec.lock (hunk 3 above)
Comment (tddang-linagora) [reply pubspec.lock]: Done

**Review by hoangdat — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Pull request merged**